news/4.39/platform_isv.md

```diff
@@ -67,3 +67,44 @@ pdf.dispose();
 
 This enhancement eliminates the need for external PDF libraries and provides a native, platform-integrated solution for PDF generation in SWT applications.
 See [Snippet388](https://github.com/eclipse-platform/eclipse.platform.swt/blob/master/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet388.java) for a complete working example.
+
+### ImageDataAtSizeProvider Referencable
+
+<details>
+<summary>Contributors</summary>
+- [Michael Bangas](https://github.com/Michael5601)
```

Comment:
A blank line after the `<summary>` is important or it won't look right:

<img width="1025" height="131" alt="Image" src="https://github.com/user-attachments/assets/068eefb8-ec7f-4da9-afb5-28c2545c29c5" />